style.css

```diff
@@ -14,8 +14,7 @@ This theme, like WordPress, is licensed under the GPL.
 Use it to make something cool, have fun, and share what
 you've learned with others.
 
-Blank Theme is based on Underscores http://underscores.me/, (C) 2012-2015 Automattic, Inc.
-Underscores is distributed under the terms of the GNU GPL v2 or later.
+Blank Theme is based on Blank theme https://github.com/rtCamp/blankTheme
```

Comment:
Blank Theme is also distributed under the same license, want to retain the license string.